sdk/vc-data/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@affinidi/vc-data",
-  "version": "1.3.7",
+  "version": "1.3.8",
```

Comment:
package.json: updated from 1.4.0 to 1.4.1.
package-lock.json: updated from 1.3.7 to 1.3.8.

So which of these is correct?

Reply:
for package.json the correct version is 1.4.1 and for package-lock.json its 1.3.8

Reply:
package-lock.json and package.json should have the same version (which is the package version)